### ISSUE TITLE
Failed to register {000204EF-0000-0000-C000-000000000046}

### ISSUE THREAD

**Issue by glennwiz** [timabell/msaccess-vcs-integration#8]
Tested on two machines and gets the same error on both of them when trying the import function from Access 2013

failed to register {000204EF-0000-0000-C000-000000000046}

**Comment by jwbrookes** [comment]
What libraries are you referencing?
That looks like the reference for VBA; if it is then it won't effect the project. The VBA library is present by default.

**Comment by jwbrookes** [comment]
I've added the error number to the message box title. if you could download the master, run the import again and tell me what the error number is then I'll look into this.

**Comment by glennwiz** [comment]
The error number is -2147319779, i have only the default refrences that is already specified in the demo.accdb

**Comment by jwbrookes** [comment]
Well, the error number is meaningless.
For each of the following lines, copy it into the immediate window in the VBA editor and hit return: 
`Debug.Print application.References("VBA").Guid`
`Debug.Print application.References("VBA").FullPath`
`Debug.Print application.References("VBA").Major`
`Debug.Print application.References("VBA").Minor`
and then post the output here.

**Comment by glennwiz** [comment]
Your suspicion that is was vba is correct.

Debug.Print application.References("VBA").Guid
{000204EF-0000-0000-C000-000000000046}
Debug.Print application.References("VBA").FullPath
C:\PROGRA~1\COMMON~1\MICROS~1\VBA\VBA7.1\VBE7.DLL
Debug.Print application.References("VBA").Major
 4 
Debug.Print application.References("VBA").Minor
 2

**Comment by Ronaldvr** [comment]
I think this may be related to the constraints error here: https://github.com/timabell/msaccess-vcs-integration/issues/16
I get the same error becasue the constraint gets a GUID as well....

**Comment by Ronaldvr** [comment]
I found an easy solution by using the "BuiltIn" property:

'''vba
    For Each ref In Application.References
        If ref.GUID > "" Then ' references of types mdb,accdb,mde etc don't have a GUID
        ElseIf Not ref.BuiltIn Then
          line = ref.FullPath
          outfile.WriteLine line
        End If
    Next

```

**Comment by Ronaldvr** [comment]
Its seems that I did not test this correctly or that this gives odd results, but when I tested it this mornig this suddely failed to do the trick. So chalk this up as a dead end.

**Comment by Ronaldvr** [comment]
The correct code is as follows:

```vba
Public Function ExportReferences(obj_path As String)
Dim FSO, outfile
Dim line As String
Dim ref As Reference

Set FSO = CreateObject("Scripting.FileSystemObject")
Set outfile = FSO.CreateTextFile(obj_path & "references.csv", True)
For Each ref In Application.References
    If ref.GUID > "" Then ' references of types mdb,accdb,mde etc don't have a GUID
        If Not ref.BuiltIn Then
            line = ref.GUID & "," & CStr(ref.Major) & "," & CStr(ref.Minor)
            outfile.WriteLine line
        End If
    Else
        line = ref.FullPath
        outfile.WriteLine line
    End If
Next
outfile.Close
End Function


```

**Comment by timabell** [comment]
fixed in bde584e3c64f69c0542177ff9829c5d42692ea34
already pushed to master.
thanks for the code @Ronaldvr
